### PR TITLE
[FIX] Add correct check for clang.... again

### PR DESCRIPTION
**Pull request by juangpc**
😮‍💨

### CODE REVIEW

File: applications/mne_anonymize/mne_anonymize.pro [hunk 1]
```diff
@@ -123,7 +123,7 @@ HEADERS += \
 FORMS += \
     mainwindow.ui
 
-*-clang++ {
+clang {
```
Comment (LorenzE) [inline applications/mne_anonymize/mne_anonymize.pro:126]: why is this needed in only this one file? Weren't other files changed as well?

**Review by LorenzE — APPROVED** [review]

File: applications/mne_anonymize/mne_anonymize.pro (hunk 1 above)
Comment (juangpc) [reply applications/mne_anonymize/mne_anonymize.pro:126]: all other 109 appearances where changed in #872 . This one escaped.
